ci-operator/jobs/openshift/release/openshift-release-master-presubmits.yaml

```diff
@@ -442,6 +442,8 @@ presubmits:
       - args:
         - --config-path
         - ./core-services/prow/02_config/_config.yaml
+        - --supplemental-prow-config-dir
+        - core-services/prow/02_config
```

Comment:
```suggestion
        - ./core-services/prow/02_config
```

To keep the consistency?

Reply:
I had that initially and that makes checkconfig fail :/